### ISSUE TITLE
get fine-tuning working better

### ISSUE THREAD

**Issue by mobeets** [mobeets/neuralnetmusic#22]

**Comment by mobeets** [comment]
okay, well, as soon as i started printing out more debug info, suddenly it decided it wanted to fine-tune for longer. magically.

there appears to be this "use_autoencoder" thing though, such that instead of trying to reproduce the inputs, the code is randomly generating labels for all of the training data and then trying to regenerate them from the labels. this seems weird.

**Comment by mobeets** [comment]
somehow fixed
